.git-blame-ignore-revs

```diff
@@ -2,3 +2,4 @@
 b42067f6776dcd763827000d585a88e071b78af3
 # applied prettier - https://github.com/glass-dev/glass/pull/227
 f9bac62f497a7288aa71fd4dbd948945c37f854e
+# applied ruff-linting - https://github.com/glass-dev/glass/pull/232
```

Comment:
@Saransh-cpp when this is merged can you add the commit hash underneath?

Reply:
🫡